packages/airnode-deployer/src/handlers/gcp/index.ts

```diff
@@ -12,8 +21,29 @@ export async function startCoordinator(_req: Request, res: Response) {
   res.status(200).send(response);
 }
 
-export async function initializeProvider(req: Request, res: Response) {
-  const stateWithConfig = { ...req.body.state, config: parsedConfig };
+export async function run(req: Request, res: Response) {
+  const payload = req.body;
+
+  if (payload.functionName === 'initializeProvider') {
```

Comment:
use TS checked switch. You can also use the `return fnCall()` as the return value is not important for this function.